public/app/core/components/Select/OrgPicker.tsx

```diff
@@ -36,7 +36,7 @@ export class OrgPicker extends PureComponent<Props, State> {
   }
 
   getOrgOptions = async (query: string): Promise<Array<SelectableValue<number>>> => {
-    if (!this.orgs) {
+    if (!this.orgs || !this.orgs.length) {
```

Comment:
```suggestion
    if (!this.orgs?.length) {
```
This should work as well I think.

Reply:
Beautiful, 🙇